cmd/pool/debug.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"path"
+	"path/filepath"
+
+	"github.com/lightninglabs/pool"
+	"github.com/lightninglabs/pool/auctioneer"
+	"github.com/lightninglabs/pool/clientdb"
+	"github.com/lightninglabs/pool/order"
+	"github.com/lightninglabs/pool/poolrpc"
+	"github.com/lightninglabs/pool/terms"
+	"github.com/lightningnetwork/lnd/lncfg"
+	"github.com/urfave/cli"
+)
+
+var debugCommands = []cli.Command{
+	{
+		Name:      "debug",
+		ShortName: "d",
+		Usage:     "Debug the local trader state.",
+		Hidden:    true,
+		Subcommands: []cli.Command{
+			dumpAccountsCommand,
+			dumpOrdersCommand,
```

Comment:
Any reason to have these given that we have `pool accounts list` and `pool orders list`?

Reply:
I should have given some more explanation, you're right. I did this in a hurry to get a fix for the bug out quickly.
The advantage of these commands is that they work without starting the trader daemon (so no interaction with the server) and also work on databases of other users, as no interaction with `lnd` is required either.
Perhaps it's really just something we'll use and it doesn't need to be in the main code base. But I've hidden the command group so most people won't notice this.